docs/pages/table.js

```diff
@@ -145,90 +75,1666 @@ card(
+            cardSize="md"
+            type="don't"
+            title="When Not to Use"
+            description={`
+          - There will never be enough data to populate at least 2 rows.
+          - Displaying content that doesn’t follow a consistent pattern and can't be broken down into columns.
+          - Providing robust data that doesn't fit in a tabular format. If there is a need to display a more complex data relationship, consider an info-graphic or a non-tabular format.
+        `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection title="Style" columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use accessible Gestalt grays for table text, and reserve colors to sparingly accent important status and information. Avoid over-styling text."
+            defaultCode={`
+function Example() {
+  const HeaderRow = ({ id }) => {
+    return (
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell>
+            <Box display="visuallyHidden"><Label htmlFor={id}>Not all checkboxes are checked</Label></Box>
+            <Checkbox
+              id={id}
+   onChange={() => {}}
+              indeterminate
+              size="sm"
+            />
+          </Table.HeaderCell>
+          {["Status", "Campaign"].map((title, idx) => {
+            return (
+              <Table.HeaderCell key={idx}>
+                <Text weight="bold">{title}</Text>
+              </Table.HeaderCell>
+            );
+          })}
+        </Table.Row>
+      </Table.Header>
+    );
+  };
+
+  const BaseRow = ({ id, checked, disabled, text, campaign }) => {
+    return (
+      <Table.Row>
+        <Table.Cell>
+          <Checkbox
+            id={id.replace(/ /g, "_").replace(/'/g, "") + "_" + text.replace(/ /g, "_").replace(/'/g, "")} onChange={() => {}}
+            disabled={disabled}
+            size="sm"
+            checked={checked}
+          />
+        </Table.Cell>
+        <Table.Cell>
+          <Label htmlFor={id.replace(/ /g, "_").replace(/'/g, "") + "_" + text.replace(/ /g, "_").replace(/'/g, "")}>
+            <Text>{text}</Text>
+          </Label>
+        </Table.Cell>
+        <Table.Cell>
+          <Text color={disabled ? "gray" : "darkGray"}>{campaign}</Text>
+        </Table.Cell>
+      </Table.Row>
+    );
+  };
+
+  const tableID = "Example of a 'do' for table style";
+
+  return (
+    <Table accessibilityLabel={tableID}>
+      <HeaderRow id={tableID}/>
+      <Table.Body>
+        <BaseRow
+          id={tableID}
+          checked={true}
+          text="The best ad"
+          campaign="Engagement"
+        />
+        <BaseRow
+          id={tableID}
+          disabled
+          text="This ad is great"
+          campaign="Awareness"
+        />
+        <BaseRow
+          id={tableID}
+          checked={true}
+          text="Macy's pincycle"
```

Comment:
I think we should avoid real company names (Macys and Best Buy), and only use fake names

Reply:
DONE